docs/ref/models/options.txt

```diff
@@ -283,7 +283,7 @@ Django quotes column and table names behind the scenes.
 
 .. attribute:: Options.default_permissions
 
-    Defaults to ``('add', 'change', 'delete')``. You may customize this list,
+    Defaults to ``('view', 'add', 'change', 'delete')``. You may customize this list,
```

Comment:
need `.. versionchanged:: 1.9` annotation at the end of this section noting that the "view" permission was added.